features/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/VERSION.txt

```diff
@@ -1 +1 @@
-1d1faca481c3
+eaaf44c87c63
```

Comment:
Please ensure that the VERSION.txt file documenting with commit hash of TF-M the binaries were built with is correct. All TF-M binaries in Mbed OS are currently are build with the same TF-M branch, but this PR provides a binary for PSoC64 different from the others.

We may want a PSoC64-specific `VERSION.txt` as well as a "permissive binary license" (if source is unavailable).

Reply:
I will update the version file, Please advise what algorithm is used to generate the hash

Reply:
The algorithm is: the first 12 characters of the git hash of the branch of TF-M you are using to make the binary